workshops/templates/workshops/person_merge_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "workshops/_page.html" %}
+
+{% load crispy_forms_tags %}
+{% load selectable_tags %}
+
+{% block content %}
+
+    {% crispy person_merge_form %}
```

Comment:
To help render forms nicely, crispy-forms uses form helper. We have one or two, so you can simply render form using that helper with this tag:

```
    {% crispy person_merge_form form_helper %}
```